pyproject.toml

```
@@ -28,7 +28,7 @@ maintainers = [
     { name = "VASim Maintainers", email = "vasim-maintainers@service.microsoft.com" },
 ]
 
-version = "0.1.0"
+version = "0.1.1"
```

```suggestion
version = "0.1.0"
```

leaving version as is for now